lib/scala/library-manager-test/src/main/scala/org/enso/librarymanager/published/repository/DummyRepository.scala

```diff
@@ -177,20 +177,22 @@ abstract class DummyRepository {
     val serverDirectory =
       Path.of("tools/simple-library-server").toAbsolutePath.normalize
 
-    val preinstallCommand =
-      commandPrefix ++ Seq(npmCommand, "install")
-    val preinstallExitCode = new ProcessBuilder()
-      .command(preinstallCommand: _*)
-      .directory(serverDirectory.toFile)
-      .inheritIO()
-      .start()
-      .waitFor()
-
-    if (preinstallExitCode != 0)
-      throw new RuntimeException(
-        s"Failed to preinstall the Library Repository Server dependencies: " +
-        s"npm exited with code $preinstallCommand."
-      )
+    if (!DummyRepository.isCI) {
```

Comment:
I'd appreciate a short comment mentioning why we can skip this (because on CI we ensure this is preinstalled earlier).